ml-agents-envs/mlagents/envs/subprocess_env_manager.py

```diff
@@ -192,19 +224,14 @@ def step(self) -> List[EnvironmentStep]:
         step_infos = self._postprocess_steps(worker_steps)
         return step_infos
 
-    def reset(
-        self,
-        config: Optional[Dict] = None,
-        train_mode: bool = True,
-        custom_reset_parameters: Any = None,
-    ) -> List[EnvironmentStep]:
+    def reset(self, config: Optional[Dict] = None) -> List[EnvironmentStep]:
         while any(ew.waiting for ew in self.env_workers):
             if not self.step_queue.empty():
                 step = self.step_queue.get_nowait()
                 self.env_workers[step.worker_id].waiting = False
         # First enqueue reset commands for all workers so that they reset in parallel
         for ew in self.env_workers:
-            ew.send("reset", (config, train_mode, custom_reset_parameters))
+            ew.send("reset", (config))
```

Comment:
nit: Don't need the () here since it's not a tuple anymore
```suggestion
            ew.send("reset", config)
```